### PR TITLE
Modularize routes to match clark-service

### PR DESCRIPTION
**Pull request by mfranz1**
# Description
This PR takes a jab at trying to resolve route issues. Through local testing, routes that were said to conflicts no longer do.

### CODE REVIEW

**Review by shanioffe — APPROVED** [review]

File: src/modules/clark/hierarchy-module/hierarchy.routes.ts [hunk 1]
```diff
@@ -7,11 +7,11 @@ export const HIERARCHY_ROUTES: ProxyRoute[] = [
      * clark-service yet and will have a different target
      * than clark-service
      *
-     * The target for the following routes shoud be  hierarchy-service
+     * The target for the following routes should be  hierarchy-service
```
Comment (pzalep1) [inline src/modules/clark/hierarchy-module/hierarchy.routes.ts:10]: 👌🏻

File: src/modules/clark/learning-object-module/change-authorship.routes.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import { HTTPMethod } from "../../../shared/types/http-method.type";
+import { ProxyRoute } from "../../../shared/types/proxy-route.type";
+
+export const CHANGE_AUTHORSHIP_ROUTES: ProxyRoute[] = [
+    {
+        method: HTTPMethod.POST,
+        path: "/users/:userId/learning-objects/:learningObjectId/change-author",
```
Comment (pzalep1) [inline src/modules/clark/learning-object-module/change-authorship.routes.ts:7]: This didn't get switched to CUID? I haven't had a chance to look.

File: src/modules/clark/learning-object-module/changelog.routes.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+export const CHANGELOG_ROUTES: ProxyRoute[] = [
+    {
+        method: HTTPMethod.GET,
+        path: "/learning-objects/:cuid/changelogs",
+        auth: true,
+    },
+    {
+        method: HTTPMethod.POST,
+        path: "/learning-objects/:cuid/changelog",
```
Comment (pzalep1) [inline src/modules/clark/learning-object-module/changelog.routes.ts:12]: Is this supposed to be changelog or changelogs? Just wondering based on the structure from the route above.

File: src/modules/clark/learning-object-module/learning-objects.routes.ts [hunk 4]
```diff
@@ -1,332 +1,57 @@
-import { envConfig } from "../../../config/env/env.driver";
-import { CLARK_SERVICE_URI } from "../../../config/global.env";
 import { HTTPMethod } from "../../../shared/types/http-method.type";
 import { ProxyRoute } from "../../../shared/types/proxy-route.type";
 
 export const LEARNING_OBJECTS_ROUTES: ProxyRoute[] = [
-    /**
-     * The following routes have not been declared by
```
Comment (pzalep1) [inline src/modules/clark/learning-object-module/learning-objects.routes.ts:8]: image

File: src/modules/clark/learning-object-module/legacy.routes.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+        auth: true,
+        target: envConfig.getUri(LEARNING_OBJECT_SERVICE_URI),
+    },
+    // @deprecated
+    {
+        method: HTTPMethod.GET,
+        path: "/users/:username/evaluations",
+        target: envConfig.getUri(LEARNING_OBJECT_SERVICE_URI),
+    },
+];
```
Comment (pzalep1) [inline src/modules/clark/learning-object-module/legacy.routes.ts:37]: Very nice.

**Review by pzalep1 — COMMENTED** [review]
Looks good just a few comments and a meme.

File: src/modules/clark/learning-object-module/learning-objects.routes.ts (hunk 4 above)
Comment (mfranz1) [reply src/modules/clark/learning-object-module/learning-objects.routes.ts:8]: 😂😂😂😂😂😂😂

File: src/modules/clark/learning-object-module/change-authorship.routes.ts (hunk 2 above)
Comment (mfranz1) [reply src/modules/clark/learning-object-module/change-authorship.routes.ts:7]: No according to the API
image

File: src/modules/clark/learning-object-module/learning-objects.router.ts [hunk 6]
```diff
@@ -1,14 +1,37 @@
...
+import { SUBMISSIONS_ROUTES } from "./submissions.routes";
+import { TOPICS_ROUTES } from "./topics.routes";
+import { LEGACY_ROUTES } from "./legacy.routes";
 
 export class LearningObjectsRouteHandler {
     public static build(): Router {
         return buildProxyRouter(
-            LEARNING_OBJECTS_ROUTES,
-            envConfig.getUri(LEARNING_OBJECT_SERVICE_URI),
+            [
```
Comment (curlyLasagna) [inline src/modules/clark/learning-object-module/learning-objects.router.ts:21]: 🎨

**Review by stef359 — APPROVED** [review]

**Review by pzalep1 — APPROVED** [review]

**Pull request merged**